libvast/src/system/signal_reflector.cpp

```diff
@@ -0,0 +1,74 @@
+//    _   _____   __________
+//   | | / / _ | / __/_  __/     Visibility
+//   | |/ / __ |_\ \  / /          Across
+//   |___/_/ |_/___/ /_/       Space and Time
+//
+// SPDX-FileCopyrightText: (c) 2022 The VAST Contributors
+// SPDX-License-Identifier: BSD-3-Clause
+
+#include "vast/system/signal_reflector.hpp"
+
+#include <caf/blocking_actor.hpp>
+
+#include <csignal>
```

Comment:
```suggestion
#include <csignal>
#include <unistd.h>
```

Otherwise this doesn't build on macOS.